### PR TITLE
[TT-4712] docs for SQL on RHEL8

### PR DESCRIPTION
**Pull request by urbanishimwe**
related to: https://tyktech.atlassian.net/browse/TT-4712

### CODE REVIEW

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 1]
```diff
@@ -117,7 +117,17 @@ sslverify=1
...
+sudo yum -q makecache -y --disablerepo='*' --enablerepo='tyk_tyk-dashboard'
+```
+
+Finally, install Tyk dashboard and Redis.
+```bash
+sudo yum install -y tyk-dashboard redis
+```
+### Step 3: Configure and Install MongoDB v4.0 or SQL
 {{< tabs_start >}}
 {{< tab_start "MongoDB" >}}
```
Comment (jeffy-mathew) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md]: shall we add a line break 
```suggestion
<br>
```

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 2]
```diff
@@ -131,45 +141,62 @@ enabled=1
...
+# Optionally ensure that MongoDB will start following a system reboot
+sudo systemctl enable mongod
+# start MongoDB server
+sudo systemctl start mongod
 ```
 {{< tab_end >}}
 {{< tab_start "SQL" >}}
 
-We recommend installing MongoDB and then using our new [SQL migration tool]({{< ref "/content/planning-for-production/database-settings/sql-configuration.md#migrating-from-an-existing-mongodb-instance" >}}).
+For the purpose of this tutorial, we'll use PostgreSQL version 13.
```
Comment (jeffy-mathew) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md:161]: shall we add a line break 
```suggestion
<br>
```

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 3]
```diff
@@ -183,13 +210,18 @@ You need to replace `<hostname>` for `--redishost=<hostname>`, and `<IP Address>
 ```bash
 sudo /opt/tyk-dashboard/install/setup.sh --listenport=3000 --redishost=<hostname> --redisport=6379 --mongo=mongodb://<IP Address>/tyk_analytics --tyk_api_hostname=$HOSTNAME --tyk_node_hostname=http://localhost --tyk_node_port=8080 --portal_root=/portal --domain="XXX.XXX.XXX.XXX"
 ```
-**ADD SQL**
 {{< note success >}}
 **Note**  
 
 Make sure to use the actual DNS hostname or the public IP of your instance as the last parameter.
 {{< /note >}}
+for **SQL** database, We recommend installing MongoDB and then using our new [SQL migration tool]({{< ref "/content/planning-for-production/database-settings/sql-configuration.md#migrating-from-an-existing-mongodb-instance" >}}).
```
Comment (jeffy-mathew) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md]: We've installation scripts to setup dashboard with SQL. Shall we update it? 
Refer - https://github.com/TykTechnologies/tyk-docs/blob/master/tyk-docs/content/tyk-on-prem/installation/debian-ubuntu/dashboard.md#prerequisites-2 

Also instructing users to setup dashboard with MongoDB first and then use migration tool doesn't seem to be a straight forward approach

**Comment by jeffy-mathew** [comment]
@urbanishimwe 
We've to update the installation instructions for tyk pump

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 4]
```diff
@@ -131,66 +141,86 @@ enabled=1
...
-### Step 4: Install Packages
 
-We're ready to go, you can now install the relevant packages using yum:
+Optionally initialize the database and enable automatic start:
 ```bash
-sudo yum install -y mongodb-org tyk-dashboard redis
+# Initialize database
+sudo /usr/pgsql-13/bin/postgresql-13-setup initdb
+# Optionally ensure that MongoDB will start following a system reboot
+sudo systemctl enable mongod
```
Comment (jeffy-mathew) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md]: We init postgresql at line [152](https://github.com/TykTechnologies/tyk-docs/pull/1863/files#diff-68db9304936cc07f438a80ead4295e44abec4735ecc785753f971aaa5e35642aR152) but the startup service mentioned is mongod

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 5]
```diff
@@ -131,66 +141,86 @@ enabled=1
...
 
-Make sure to use the actual DNS hostname or the public IP of your instance as the last parameter.
-{{< /note >}}
+{{< tab_end >}}
+{{< tabs_end >}}
 
 
+You need to replace `<hostname>` for `--redishost=<hostname>`, and `<IP Address>` for `--mongo=mongodb://<IP Address>/` with your own values to run this script.
+
 What we have done here is:
```
Comment (jeffy-mathew) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md:163]: We've to add description for 
`--storage=postgres --connection_string="host=<Postgres Host Name>` here

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 6]
```diff
@@ -131,66 +141,86 @@ enabled=1
...
+```bash
+sudo /opt/tyk-dashboard/install/setup.sh --listenport=3000 --redishost=<hostname> --redisport=6379 --mongo=mongodb://<IP Address>/tyk_analytics --tyk_api_hostname=$HOSTNAME --tyk_node_hostname=http://localhost --tyk_node_port=8080 --portal_root=/portal --domain="XXX.XXX.XXX.XXX"
+```
 
+{{< tab_end >}}
+{{< tab_start "SQL" >}}
 
 ```bash
-sudo /opt/tyk-dashboard/install/setup.sh --listenport=3000 --redishost=<hostname> --redisport=6379 --mongo=mongodb://<IP Address>/tyk_analytics --tyk_api_hostname=$HOSTNAME --tyk_node_hostname=http://localhost --tyk_node_port=8080 --portal_root=/portal --domain="XXX.XXX.XXX.XXX"
+sudo /opt/tyk-dashboard/install/setup.sh --listenport=3000 --redishost=<hostname> --redisport=6379 --storage=postgres --connection_string="host=<Postgres Host Name> port=<Port> user=<User> password=<Password> dbname=<DB>" --tyk_api_hostname=$HOSTNAME --tyk_node_hostname=http://localhost --tyk_node_port=8080 --portal_root=/portal --domain="XXX.XXX.XXX.XXX"
```
Comment (jeffy-mathew) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md:155]: A note, similar to https://github.com/TykTechnologies/tyk-docs/pull/1863/files#diff-39173e0c39373600f7c6e8897041ef2b43efde72d907b024c53e5dfaaeeeced9R136 can be useful here

**Review by jeffy-mathew — APPROVED** [review]

**Review by lghiur — APPROVED** [review]

**Review by marksou — APPROVED** [review]
👍

File: tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md [hunk 7]
```diff
@@ -127,74 +127,14 @@ Finally, install Tyk dashboard and Redis.
...
-### Step 4: Start MongoDB, PostgreSQL and Redis
+### Step 3: Start MongoDB, PostgreSQL and Redis
 
 In many cases MongoDB/SQL or Redis might not be running. start redis:
 ```bash
 sudo service redis start
 ```
-**check step 3, on how to start MongoDB or PostgreSQL**
-### Step 5: Configure Tyk Dashboard
+**check Getting started on Red Hat (RHEL / CentOS), on how to start MongoDB or PostgreSQL**
```
Comment (urbanishimwe) [inline tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md]: This should be a link, or we could keep the above deleted docs

Comment (jeffy-mathew) [reply tyk-docs/content/tyk-on-prem/installation/redhat-rhel-centos/dashboard.md]: done

**Pull request merged**